assets/queries/template/metadata.json

```diff
@@ -4,5 +4,6 @@
   "severity": "HIGH",
   "category": null,
   "descriptionText": "<TEXT>",
-  "descriptionUrl": "#"
-}
\ No newline at end of file
+  "descriptionUrl": "#",
+  "platform": "template"
```

Comment:
Wouldn't be better if it was &lt;PLATFORM&gt; instead of template?

Reply:
resolved